jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/extensions/compress/PerMessageDeflateExtension.java

```diff
@@ -70,6 +71,12 @@ public void incomingFrame(Frame frame)
             nextIncomingFrame(frame);
             return;
         }
+
+        if (frame.getOpCode() == OpCode.CONTINUATION && frame.isRsv1())
+        {
+            // Per RFC7692 we MUST Fail the websocket connection
+            throw new ProtocolException("Invalid RSV1 set on permessage-deflate CONTINUATION frame");
```

Comment:
@lachlan-roberts Looks like @joakime has decided to fail the connection in 9.4 (as per the RFC) , so I think you need to change 10 to also fail rather than ignore/warn.